wire-ios/Wire-iOS Tests/ConversationList/Container/ConversationListViewControllerViewModelTests.swift

```diff
@@ -24,13 +24,15 @@ final class ConversationListViewControllerViewModelTests: XCTestCase {
     private var mockViewController: MockConversationListContainer!
     private var selfUser: MockUserType!
     private var mockConversation: ZMConversation!
+    private var userSession: UserSessionMock!
 
     override func setUp() {
         super.setUp()
 
         let account = Account.mockAccount(imageData: Data())
         selfUser = .createSelfUser(name: "Bob")
-        sut = ConversationListViewController.ViewModel(account: account, selfUser: selfUser)
+        userSession = UserSessionMock()
```

Comment:
Also here we should pass in the self user to the user session.

Reply:
Done!